plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js

```diff
@@ -88,8 +88,10 @@ function createFile(currentTaskName,state) { //createFile
             htconfigDom = marshalEditorTextContent(data);
             generateHTConfig(htconfigDom, xmlDom, currentTaskName);
             saveHTConfig(htconfigDom);
-            generateUI();
-            if(state=="initial"){
+            if(state != "generateText"){
```

Comment:
need space between if and brackets and between closing bracket and curly brackets.